### PR TITLE
Added scrapping data from tags with single quotes

### PR DESCRIPTION
**Pull request by PVGrad**
Some sites has such markup that represents some tags with signle quotes instead of double (i.e. imdb.com), so I've changes regexps to be ready for them

### CODE REVIEW

**Comment by tomaj** [comment]
Thank you! looks ok 👍  But can you please update tests and include this functionality there? 🙏

**Comment by PVGrad** [comment]
> Thank you! looks ok 👍 But can you please update tests and include this functionality there? 🙏

done. check, please.

File: tests/ScraperTest.php [hunk 1]
```diff
@@ -219,4 +225,38 @@ public function testMerge()
         $this->assertEquals(new \DateTime('@' . strtotime('2018-12-17T00:28:43+00:00')), $meta->getModifiedTime());
 
     }
+    public function testSingleQuotedMetaProperty()
+    {
+        $data = <<<EOT
+            <title>Page title</title>
+            <meta name='description' content="Default page description"/>
```
Comment (tomaj) [inline tests/ScraperTest.php]: What about putting single quotes also into `content`?

Comment (PVGrad) [reply tests/ScraperTest.php]: done.

**Pull request merged**